.github/workflows/ci_build.yml

```diff
@@ -94,7 +94,12 @@ jobs:
       uses: actions/checkout@v1
     - name: Run build script
       run: |
-        python scripts\build.py --skip-check-code-style --skip-tests --config ${{ matrix.config.type }}
+        if [ "${{ matrix.config.type }}" == "release" ]
+        then
+          python scripts\build.py --skip-check-code-style --skip-tests --config ${{ matrix.config.type }}
```

Comment:
If including tests here would put test binaries in the released packages, it makes sense not to enable them.